mmpose/datasets/datasets/top_down/topdown_onehand10k_dataset.py

```diff
@@ -0,0 +1,244 @@
+            Default: None.
+        data_cfg (dict): config
+        pipeline (list[dict | callable]): A sequence of data transforms.
+        test_mode (bool): Store True when building test or
+            validation dataset. Default: False.
+    """
+
+    def __init__(self,
+                 ann_file,
+                 img_prefix,
+                 data_cfg,
+                 pipeline,
+                 test_mode=False):
+
+        super().__init__(
+            ann_file, img_prefix, data_cfg, pipeline, test_mode=test_mode)
+
+        self.ann_info['flip_pairs'] = []
+
+        self.ann_info['use_different_joints_weight'] = False
+        self.ann_info['joints_weight'] =  \
+            np.ones(21, dtype=np.float32).reshape(
+                (self.ann_info['num_joints'], 1))
+
+        self.db = self._get_db(ann_file)
+        self.image_set = set([x['image_file'] for x in self.db])
+        self.num_images = len(self.image_set)
+
+        print('=> num_images: {}'.format(self.num_images))
+        print('=> load {} samples'.format(len(self.db)))
+
+    def _get_db(self, ann_file):
+        """Load dataset."""
+        with open(ann_file, 'r') as f:
+            data = json.load(f)
+        tmpl = dict(
+            image_file=None,
+            center=None,
+            scale=None,
+            rotation=0,
+            joints_3d=None,
+            joints_3d_visible=None,
+            dataset='OneHand10K')
+
+        imid2info = {x['id']: x for x in data['images']}
+
+        num_joints = self.ann_info['num_joints']
+        gt_db = []
+
+        for anno in data['annotations']:
+            newitem = cp.deepcopy(tmpl)
+            image_id = anno['image_id']
+            newitem['image_file'] = os.path.join(
+                self.img_prefix, imid2info[image_id]['file_name'])
+
+            if max(anno['keypoints']) == 0:
+                continue
+
+            joints_3d = np.zeros((num_joints, 3), dtype=np.float)
+            joints_3d_visible = np.zeros((num_joints, 3), dtype=np.float)
+
+            for ipt in range(num_joints):
+                joints_3d[ipt, 0] = anno['keypoints'][ipt * 3 + 0]
+                joints_3d[ipt, 1] = anno['keypoints'][ipt * 3 + 1]
+                joints_3d[ipt, 2] = 0
+                t_vis = anno['keypoints'][ipt * 3 + 2]
+                if t_vis > 1:
+                    t_vis = 1
+                joints_3d_visible[ipt, 0] = t_vis
+                joints_3d_visible[ipt, 1] = t_vis
+                joints_3d_visible[ipt, 2] = 0
+
+            center, scale = self._box2cs(anno['bbox'][:4])
+            newitem['center'] = center
+            newitem['scale'] = scale
+            newitem['joints_3d'] = joints_3d
+            newitem['joints_3d_visible'] = joints_3d_visible
+            gt_db.append(newitem)
+
+        return gt_db
+
+    def _box2cs(self, box):
+        """Get box center & scale given box (x, y, w, h)."""
+        x, y, w, h = box[:4]
+        return self._xywh2cs(x, y, w, h)
```

Comment:
`self._xywh2cs(*box[:4])`

Reply:
maybe we can remove this helper function, since it is a one-liner

Reply:
Github action will check the coverage of the code-comments, so let's keep the helper function.

Reply:
what's the relationship between coverage and removing the function...

Reply:
Ahaha. Sorry, I misunderstood it. 
Yes. Just remove this function. `_xywh2cs` is enough.